Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.6-slim
+
+WORKDIR /bot
+
+COPY requirements.txt \
+        setup.py \
+        LICENSE \
+        README.md \
+        /bot/
+COPY dorpsgek_github /bot/dorpsgek_github
+
+RUN pip install -r requirements.txt
+RUN pip install -e .
```

Comment:
why -e step?

Reply:
To install the local package too. This makes things like `entry_points` available.